sorted_words.rb

```diff
@@ -0,0 +1,9 @@
+user_words = []
+loop do 
+puts "Add whatever words you would like! (When finsished adding words, simply leave blank and hit enter)"
+user_input = gets.chomp
+break if user_input == ''
+user_words.push(user_input)
+end
```

Comment:
Similar to my indentation suggestion for methods on klove2016/Backend-sqlruby#1, you should indent multi-line blocks (`do ... end`) as well.

```suggestion
loop do 
  puts "Add whatever words you would like! (When finsished adding words, simply leave blank and hit enter)"
  user_input = gets.chomp
  break if user_input == ''
  user_words.push(user_input)
end
```